packager/import.go

```diff
@@ -0,0 +1,91 @@
+package packager
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/deislabs/cnab-go/bundle"
+	"github.com/deislabs/cnab-go/bundle/loader"
+	"github.com/docker/docker/pkg/archive"
+)
+
+var (
+	// ErrNoArtifactsDirectory indicates a missing artifacts/ directory
+	ErrNoArtifactsDirectory = errors.New("No artifacts/ directory found")
+)
+
+// Importer is responsible for importing a file
+type Importer struct {
+	Source      string
+	Destination string
+	Loader      loader.BundleLoader
+	Verbose     bool
+}
+
+// NewImporter creates a new secure *Importer
+//
+// source is the filesystem path to the archive.
+// destination is the directory to unpack the contents.
+// load is a loader.BundleLoader preconfigured for loading bundles.
+func NewImporter(source, destination string, load loader.BundleLoader, verbose bool) (*Importer, error) {
+	return &Importer{
+		Source:      source,
+		Destination: destination,
+		Loader:      load,
+		Verbose:     verbose,
+	}, nil
+}
+
+// Import decompresses a bundle from Source (location of the compressed bundle) and properly places artifacts in the correct location(s)
+func (im *Importer) Import() error {
+	_, _, err := im.Unzip()
+
+	// TODO: https://github.com/deislabs/duffle/issues/758
```

Comment:
Should we add an issue in cnab-go instead to track it?

Reply:
should i just move that duffle issue over here?

Reply:
Moved.